### PR TITLE
Company no sql

### PR DESCRIPTION
**Pull request by loayelzobeidy**
the company profile

### CODE REVIEW

**Comment by mostafa-abdullah** [comment]
Can we review and merge #4 first, because this one is branched from there.

File: nosql/JobListing.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "jobID":"Integer",
```
Comment (Nabila63Ahmed) [inline nosql/JobListing.json]: You can make it a number to be consistent with all Ids types

File: nosql/JobListing.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "industryType":"String",
+  "employmentType":"String",
+  "jobFunctions":"String",
+  "positionName":"String",
+  "professionLevel":"String",
+  "companyID":"Number",
+  "companyName":"String",
+  "companyLocation":"String",
+  "compnayPicture":"URL/String",
```
Comment (Nabila63Ahmed) [inline nosql/JobListing.json]: Url turned out not to be a type :D keep the string

File: nosql/JobListing.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "relatedConnections":[{},{}],
```
Comment (Nabila63Ahmed) [inline nosql/JobListing.json]: since the related connections are users/ ids. it's a list of numbers --> ["Number"]

File: nosql/JobListing.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  "jobBrief":"String",
+  "requiredSkills":["String","String"],
```
Comment (Nabila63Ahmed) [inline nosql/JobListing.json]: ["String"] can be enough if we are just listing the names but maybe if names weren't unique an id referencing skills in SQL will be more handy --> ["Number"] , What do you think?

File: nosql/JobListing.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  "note":"String",
+  "similarJobs":[{},{},{}]
```
Comment (Nabila63Ahmed) [inline nosql/JobListing.json]: similar to connections. but here you'll be referencing job ids --> ["Number"]

**Comment by Nabila63Ahmed** [comment]
@loayelzobeidy you can apply the same notes to the second company file as well :+1:

**Pull request merged**